utils/frame/generate-bags/node-runtime/src/main.rs

```diff
@@ -0,0 +1,39 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Make the set of bag thresholds to be used with pallet-bags-list.
+
+use generate_bags::generate_thresholds;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+	/// How many bags to generate.
+	#[structopt(long, default_value = "200")]
+	n_bags: usize,
+
+	/// Where to write the output.
+	output: PathBuf,
+}
+
+fn main() -> Result<(), std::io::Error> {
+	let Opt { n_bags, output } = Opt::from_args();
+	let mut ext = sp_io::TestExternalities::new_empty();
```

Comment:
I'm not sure if it is a good example, the library is querying the total issuance, which is probably `0` when you use an empty test externalities.
I think it should be written that this setup probably needs some tweak to be accurate.

Reply:
yeah this is fair point. in reality, as long as total issuance is less than `u64::MAX`, we are fine. I will switch to CLI args though for correctness is situations

Reply:
2ee478cd6f0250994acfaf2dc0ab947a8926e91c